MediaPlayer/MediaPlayer.module.less

```diff
@@ -0,0 +1,3 @@
+.controlsFrame {
```

Comment:
Is this supposed to be here?

Reply:
I don't see any use for it. I removed it.